crossdock/client/outboundttl/behavior.go

```diff
@@ -49,13 +49,13 @@ func Run(t crossdock.T) {
 	}, nil)
 	fatals.Error(err, "expected a failure for timeout")
 
-	if err, ok := err.(transport.BadRequestError); ok {
+	if transport.IsBadRequestError(err) {
 		t.Skipf("sleep/raw method not implemented: %v", err)
 		return
 	}
 
 	form := strings.HasPrefix(err.Error(), `timeout for procedure "sleep/raw" of service "yarpc-test" after`)
 	assert.True(form, "error message has expected prefix for timeouts, got %q", err.Error())
-	_, ok := err.(transport.TimeoutError)
-	assert.True(ok, "error should be a TimeoutError, got %T", err)
+	assert.True(
+		transport.IsTimeoutError(err), "error should be a TimeoutError, got %T", err)
```

Comment:
oh whoah, on the main view, this doesn't look messed up, but when you go to "Filed changed", it's gnarly.